magefile.go

```diff
@@ -0,0 +1,10 @@
+//+build mage
+
+package main
+
+import "github.com/magefile/mage/sh"
+
+// Runs go fmt
+func Format() error {
+	return sh.Run("go", "fmt", "./...")
```

Comment:
At this moment, this is not returning an error even when formatting files, so Travis won't be notified. I'm currently investigating why